### PR TITLE
N27: Soldank

### PR DESCRIPTION
**Pull request by smokku**
_Part of #818_

### CODE REVIEW

File: content/news/027/index.md [hunk 1]
```diff
@@ -88,25 +88,27 @@ small update to improve the UI and fix a few bugs.
...
+is an open source clone of [Soldat] engine. It aims for full compatibility
+with original game files, mods and gameplay with modernized
 graphics engine and multiplayer networking code.
 
 Recent developments include:
 
-- Engine/Game code split
-- Command Line Interface
+- Engine/game code split
+- Command line interface
```
Comment (smokku) [inline content/news/027/index.md:118]: Command Line Interface (capitalized) is a reference to a well-known `CLI` acronym

Comment (17cupsofcoffee) [reply content/news/027/index.md:118]: Sure, I can change it back.

File: content/news/027/index.md [hunk 2]
```diff
@@ -102,6 +102,36 @@ small update to improve the UI and fix a few bugs.
...
+[Soldank] ([GitHub](https://github.com/smokku/soldank),
+[Discord](https://discord.gg/cTaC4UtqE6)) by [@smokku](https://twitter.com/smokkku)
+is an open source clone of [Soldat] engine. It aims for full compatibility
+with original game files, mods and gameplay with modernized
+graphics engine and multiplayer networking code.
+
+Recent developments include:
+
+- Engine/game code split
+- Command line interface
```
Comment (17cupsofcoffee) [inline content/news/027/index.md:118]:
```suggestion
- Command Line Interface
```

File: content/news/027/index.md [hunk 3]
```diff
@@ -132,6 +132,24 @@ The developers have also written a blog post:
 [Soldat]: http://soldat.pl/
 [Rhai]: http://rhai.rs/
 
+### [Graph Game]
```
Comment (smokku) [inline content/news/027/index.md:135]: I'm not sure this is the correct PR for this change ;-D

Comment (17cupsofcoffee) [reply content/news/027/index.md:135]: This is just showing up cause I resolved conflicts, if you go to https://github.com/rust-gamedev/rust-gamedev.github.io/pull/825/files the diff looks right 😅

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**